data/sensorCalibMatrices/matrix_SN282.txt

```diff
@@ -1,39 +1,39 @@
-1
-FFFC
-2
-FFF5
-B
-10000
-FFF4
-FFFF
-4
-9
-7
-5
-10000
-7
-1
-FFFF
-10000
-FFFD
-FFDB
-FFBD
-170
-FFE7
-FFD9
-FEEF
-FFFA
-13D
-FF2F
-1C
-FFFD
-FFC6
-31A
-FFF0
-FFCF
-349
-333
-6C
+7fff
+0
+0
+0
+0
+0
+0
+7fff
+0
+0
+0
+0
+0
+0
+7fff
+0
+0
+0
+0
+0
+0
+7fff
+0
+0
+0
+0
+0
+0
+7fff
+0
+0
+0
+0
+0
+0

```

Comment:
instead of changing all these calibration matrices to an identity you should just use the identity in the params file and leave this to the calibration they had or erase them if they are invalid